app/views/hub/organizations/edit.html.erb

```diff
@@ -0,0 +1,31 @@
+<% @title = @organization.name %>
+<% content_for :page_title, @title %>
+<% content_for :card do %>
+  <div class="hub-form">
+    <%= link_to t("general.all_organizations"), hub_organizations_path, class: "button button--small" %>
+
+    <%= form_with model: @organization, url: hub_organizations_path, method: :post, local: true, builder: VitaMinFormBuilder do |f| %>
+      <h1 class="form-card__title">
+        <%= @title %>
+      </h1>
+
+      <%= f.cfa_input_field :name, t("general.name") %>
```

Comment:
This section looks repeated between new and edit -- it might be worth pulling this out into a partial so that we make sure we're setting up the coalition select the same way, but its not blocking.

Reply:
Good eye. I lean toward duplicating it if it's just these two places, since it's two lines, but I can't say I really thought about it before you mentioned it.